### ISSUE TITLE
Deprecated and not used variables/arguments

### ISSUE THREAD

**Issue by vinicvaz** [EthoML/VAME#21]
The following variables are not used by the new functions but are still defined in the function definitions. Can we remove them?
- [x] `files` argument  not being used on [create_cohort_community_bag](https://github.com/EthoML/VAME/blob/6247cab2403f22acfea116a11ed9e882fc0c6b39/src/vame/analysis/community_analysis.py#L339)
- [x] `files` variable not being used on [umap_vis](https://github.com/EthoML/VAME/blob/6247cab2403f22acfea116a11ed9e882fc0c6b39/src/vame/analysis/umap_visualization.py#L23)
- [x] `file` argument not being used on [umap_label_vis](https://github.com/EthoML/VAME/blob/6247cab2403f22acfea116a11ed9e882fc0c6b39/src/vame/analysis/umap_visualization.py#L42)
- [x] `file` argument  not being used on [umap_vis_comm](https://github.com/EthoML/VAME/blob/6247cab2403f22acfea116a11ed9e882fc0c6b39/src/vame/analysis/umap_visualization.py#L64)
- [x] `anchor_1` argument  not being used on [plot_check_parameter](https://github.com/EthoML/VAME/blob/6247cab2403f22acfea116a11ed9e882fc0c6b39/src/vame/model/create_training.py#L54)
- [x] `anchor_2` argument  not being used on [plot_check_parameter](https://github.com/EthoML/VAME/blob/6247cab2403f22acfea116a11ed9e882fc0c6b39/src/vame/model/create_training.py#L54)
- [x] `num_features` argument not being used on [traindata_aligned](https://github.com/EthoML/VAME/blob/6247cab2403f22acfea116a11ed9e882fc0c6b39/src/vame/model/create_training.py#L136)
- [x] `hidden_size_layer_2` argument not being used on [rnn_model Lambda](https://github.com/EthoML/VAME/blob/6247cab2403f22acfea116a11ed9e882fc0c6b39/src/vame/model/rnn_model.py#L69)
- [x] `epoch` argument not being use on [test](https://github.com/EthoML/VAME/blob/6247cab2403f22acfea116a11ed9e882fc0c6b39/src/vame/model/rnn_vae.py#L271)
- [x] `optimizer` argument not being used on [test](https://github.com/EthoML/VAME/blob/6247cab2403f22acfea116a11ed9e882fc0c6b39/src/vame/model/rnn_vae.py#L271)

**Comment by bendichter** [comment]
@katiekly are you familiar with these functions? What do you think?

**Comment by katiekly** [comment]
Hi Ben, I'm not familiar with the functions except `create_cohort_community_bag()`. If the length of  `labels` equals the # of sessions then that might be why 'files' isn't be used.  

`num_features` = # of annotation points * 2. We should be using all the annotation points in the training data. 

Let me loop in Kevin and Pavol as they will have better insight whether we should remove these.
